include/wx/colordlg.h

```diff
@@ -31,6 +31,52 @@
     #define wxColourDialog wxGenericColourDialog
 #endif
 
+// Under some platforms (currently only wxMSW) wxColourDialog can send events
+// of this type while it is shown.
+//
+// Notice that this class is almost identical to wxColourPickerEvent but it
+// doesn't really sense to reuse the same class for both controls.
```

Comment:
Perhaps "really sense" should be changed to "really make sense"?

Reply:
Yes, of course, thanks for noticing!